libcudacxx/include/cuda/std/__tuple_dir/make_tuple_types.h

```diff
@@ -48,18 +48,18 @@ struct __make_tuple_types_flat<_Tuple<_Types...>, __tuple_indices<_Idx...>>
   using __tuple_types_list = __type_list<_Types...>;
 
   // Specialization for pair, tuple, and __tuple_types
-  template <class _Tp, class _ApplyFn = __apply_cv_t<_Tp>>
+  template <class _Tp, class _ApplyFn = __apply_cv_fn<_Tp>>
```

Comment:
In libcu++ we commonly use `meow` for the trait and `meow_t` for the alias

I would need a strong argument to change that

Reply:
`__apply_cv_fn` is neither a trait nor an alias. it is a metafunction.